spec/models/feedback_reporters/feedback_reporter_spec.rb

```diff
@@ -56,38 +56,38 @@
     end
 
     it "calls the ZohoResourceClient with the expected arguments" do
-      expect(ZohoResourceClient).to receive(:new).
-        with(access_token: "x7y8z9", email: "walrus@example.org")
+      expect(ZohoResourceClient).to receive(:new)
+        .with(access_token: "x7y8z9", email: "walrus@example.org")
 
       subject.send_report!
     end
 
     it "calls the Zoho ticket creator with the expected arguments" do
-      expect(ZohoResourceClient).to receive_message_chain(:new, :create_ticket).
-        with(ticket_attributes: expected_ticket_attributes)
+      expect(ZohoResourceClient).to receive_message_chain(:new, :create_ticket)
+        .with(ticket_attributes: expected_ticket_attributes)
 
       subject.send_report!
     end
   end
 
   describe "#send_attachment!" do
-    let(:work_id) { "123" }
     let(:download) { "the_file" }
 
     let(:expected_attachment_attributes) do
-      { file: download }
+      { file: StringIO.new(download) }
     end
```

Comment:
Is this used anywhere?

Reply:
Nope! I had been experimenting with other ways of matching arguments, but `StringIO` doesn't play very nicely with equality checks